extensions/amp-twitter/validator-amp-twitter.protoascii

```diff
@@ -30,10 +30,47 @@ tags: {  # <amp-twitter>
   html_format: AMP
   tag_name: "AMP-TWITTER"
   requires_extension: "amp-twitter"
+  # Tweets
   attrs: {
     name: "data-tweetid"
-    mandatory: true
+    mandatory_oneof: "['data-tweetid', 'data-momentid']"
   }
+  attrs: {
+    name: "data-conversation"
```

Comment:
I discussed this with @aghassemi offline, and the conclusion was that it may be better to just pass along the value through to Twitter. That way, if they add a new supported value, then we do not need to be in the loop at all.

Reply:
sgtm